content/neurodiversity-lessonbank/community_psychology-diversity/index.md

```diff
@@ -0,0 +1,112 @@
+
+Reflect on how you see or identify yourself in relation to these dimensions. Which groups do you identify with? Have you heard the phrase "check your privilege"? Have you thought of intersectionality and privilege in relation to yourself? FORRT has developed a tool which allows everyone in academia to reflect on their privilege. Look at the video below, in which Bethan Iley, one of the members of FORRT explains the concept.
+
+
+**Would you like to try the wheel?**
+
+> For [#DisabilityPrideMonth](https://twitter.com/hashtag/DisabilityPrideMonth?src=hash&ref_src=twsrc%5Etfw) here’s our new preprint on [#Neurodiversity](https://twitter.com/hashtag/Neurodiversity?src=hash&ref_src=twsrc%5Etfw) and Open Scholarship! We look at [#ND](https://twitter.com/hashtag/ND?src=hash&ref_src=twsrc%5Etfw), intersectionality, [#SocialJustice](https://twitter.com/hashtag/SocialJustice?src=hash&ref_src=twsrc%5Etfw) & [#UniversalDesign](https://twitter.com/hashtag/UniversalDesign?src=hash&ref_src=twsrc%5Etfw). Sharing lived experiences & solutions for more equitable inclusion of ND people in academia!  
+> 👉[https://t.co/FcQoppTbvD](https://t.co/FcQoppTbvD) [pic.twitter.com/vhF0vxy4xI](https://t.co/vhF0vxy4xI)
+> 
+> — FORRT @FORRT@mastodon.social (@FORRTproject) [July 14, 2022](https://twitter.com/FORRTproject/status/1547570498879442945?ref_src=twsrc%5Etfw)
+
+**How to use the wheel:** Look at the dimensions of privilege on the outside of the circle and choose the option from the inner circle that describes your situation best. For every answer in the smallest circle (the closest to the middle of the wheel), give yourself 1 point, then 2 points for the option in a slightly bigger circle and finally 3 points of the option closest to the outside of the circle. Share your result with a friend or keep it to yourself. Do you think this tool is useful? Is it important to check our privilege? How can this make you more mindful of other’s experiences?
+
+
+**Reading:**
+
+(to be be done before or after the lesson)
+
+Jason, Leonard A.; Glantsman, Olya; O'Brien, Jack F.; and Ramian, Kaitlyn N., "Introduction to Community Psychology: Becoming an Agent of Change" (2019). College of Science and Health Full Text Publications. 1. https://via.library.depaul.edu/cshtextbooks/1
+
+Flavio Azevedo, Sara Middleton, Jenny Mai Phan, Steven Kapp, Amélie Gourdon-Kanhukamwe, Bethan Iley, Mahmoud Elsherif, & John J. Shaw. Navigating Academia as Neurodivergent Researchers: Promoting Neurodiversity Within Open Scholarship. APS Observer October 2022
+
+Elsherif, M. M., Middleton, S. L., Phan, J. M., Azevedo, F., Iley, B. J., Grose-Hodge, M., … Dokovova, M. (2022, June 20). Bridging Neurodiversity and Open Scholarship: How Shared Values Can Guide Best Practices for Research Integrity, Social Justice, and Principled Education.
+
+Footer © Magdalena Grose-Hodge for FORRT (Framework of Open and Reproducible Research Training) About ND/Community Psychology - Diversity at main · MSGrose-Hodge/ND
```

Comment:
I think that footer line can be removed from all the pages, it is irrelevant as author is acknowledged throughout the pages